astropy/io/fits/diff.py

```diff
@@ -134,9 +135,9 @@ def identical(self):
         return not any(getattr(self, attr) for attr in self.__dict__
                        if attr.startswith('diff_'))
 
-    def report(self, fileobj=None, indent=0, clobber=False):
-        """
-        Generates a text report on the differences (if any) between two
+    @deprecated_renamed_argument('clobber', 'overwrite', '1.3')
+    def report(self, fileobj=None, indent=0, overwrite=False):
+        """Generates a text report on the differences (if any) between two
```

Comment:
As far as I'm aware, it doesn't change the overall rendering (both through Sphinx or when using the help in the interpreter); I just went with PEP257. (The same goes for blank lines at the end of a docstring, which often come about from emacs' fill-paragraph). I've changed it to the astropy convention.